SwrveSDK/src/main/java/com/swrve/sdk/SwrveBase.java

```diff
@@ -1067,7 +1071,6 @@ protected void _queueConversationEventsCommitedByUser(SwrveConversation conversa
                 }
                 queueEvent("event", parameters, payload, false);
             }
-            saveCampaignSettings();
```

Comment:
There's a method before this called _messageWasShownToUser. Does it need to have saveCampaignSettings call removed?